RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
-
-- The slideshow will now be more responsive when changing the picture
-  backward and forward.
-
-- A new module 'basic adjustments' has been added. It allows to adjust
+- A new 'basic adjustments' module has been added. It allows to adjust
   the exposure, highlight compression, contrast, brightness and saturation.
   It also has an auto feature based on Rawtherapee's auto levels that can
   work on the entire image or a user selected area.
-  Both the contrast and the auto feature use the middle grey point, the
-  default for this setting is based on the work profile.
-
-- A new module 'rgb curve' has been added. It has linked and
-  independent RGB channels.
-  It works with the rgb color space selected on the work profile.
-  The curves and histogram can be displayed using the work profile or
-  can be scaled to be 50% grey.
-  The color picker allows to select by area with <kbd>Ctrl+click</kbd>.
-  The second color picker add 4 nodes to the curve based on the area
-  selected from the image: min, average, middle, max.
-  When pressing the shortcut 'pan&zoom while editing masks' the draw area
-  can be zoomed.
-  Different types of curves can be selected.
-
-- A new module 'rgb levels' has been added. It has linked and
-  independent channels, preserve colors option and is native RGB.
 
-- A search text has been added to the module groups in the
-  darkroom. It has a configuration option to show only the modules
-  groups, only the search text or both.  Modules are searched by name
-  (localized).
+- A new 'rgb curve' module has been added. It has modes for linked and
+  independent RGB channels. The curves and histogram can be displayed using
+  the working profile or can be scaled to be 50% grey. A secondary color
+  picker will add 4 nodes to the curve based on the area selected in the
+  image: min, average, middle, and max.
+
+- A new 'rgb levels' module has been added. It has linked and
+  independent channels, preserve colors option and is native RGB.
 
-  If displaying only the search text, the active pipe modules are
-  displayed when the search text is empty.  A darkroom shortcut can be
-  set to focus on the search box.
+- A new search box has been added to the module groups in the darkroom
+  view, with configuration options to show only the module groups, only
+  the search box, or both. Modules are searched for by (localized) name,
+  and when displaying only the search box, the active modules are shown
+  when the box is empty. A shortcut can be set for focusing the box.
 
 - A preview window has been added to the darkroom that displays the edited image
   on a separate window.
 
 - A new option 'skip' is added to the 'on conflict' setting on the export module
-  that skips the export if the file already exists.
+  which skips the exporting to existing destination files.
 
-- Allow to change algorithm clone/heal and blur/color in retouch module.
+- Allow to select between clone/heal and blur/color modes in 'retouch' module.
```

Comment:
I think "change" is better for clarity, because it allows you to change from one to the other during the edit. Selection was always possible.

Reply:
Thanks, I haven't really used that feature. I'll try "switch", since I think that conveys that kind of action better?

Reply:
You can find the PR here..
https://github.com/darktable-org/darktable/pull/2360